app/views/users/show.html.erb

```diff
@@ -52,6 +52,10 @@
                             <td><strong><%= t('users.terms_and_conditions') %></strong></td>
                             <td><%= link_to t('.review_terms_and_conditions'), Setting.terms_and_conditions_link %></td>
                         </tr>
+                        <tr>
+                            <td><strong><%= t('users.daily_summary') %></strong></td>
+                            <td><%= @user.daily_summary? ? I18n.t('yes') : I18n.t('no')%></td>
```

Comment:
Check above comments. Also avoid cluttering global namespace with such values as `yes` and `no`.

Reply:
These values were already present in `et.yml` beforehand, so thought I would simply use them to provide better context. It would provide "true" and "false" to et users without it, which seems like "translation error" to me.